clang-tidy-checks/setup.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+function info() {
+  echo "[info]: " $1
+}
+
+function success() {
+  echo "success!"
+}
+
+function check_requirements() {
+  info "checking requirements"
+  cmake --version &&
+  gcc --version &&
+  python3 --version &&
+  ninja --version &&
+  lld --version &&
```

Comment:
(see above comment about `set -e`)
```suggestion
  cmake --version
  gcc --version
  python3 --version
  ninja --version
  lld --version
```